docs/src/pages/api-reference/server.mdx

```diff
@@ -6,7 +6,7 @@ The helper function to create an UploadThing instance. MAKE SURE YOU IMPORT IT
 FROM THE RIGHT PLACE. The export name ensures your file routes' `middleware`
 functions are typed correctly.
 
-<Tabs items={["Next App Router", "Next Pages Dir", "SolidJS", "Express", "Fastify", "H3"]}>
+<Tabs items={["Next App Router", "Next Pages Dir", "SolidJS", "Effect HTTP","Express", "Fastify", "H3"]}>
```

Comment:
`Effect Platform`? idk, ask the other folk over at effect how they wanna officiate this